services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -76,6 +77,9 @@ public MigrationProject getMigrationProject(Long id)
     @Override
     public MigrationProject createMigrationProject(MigrationProject migrationProject)
     {
+        if (null != getProjectIdByName(migrationProject.getTitle()))
```

Comment:
Lincoln used to use this style, too. I don't like it very much personally, as it doesn't really add anything in Java. For better or worse, we have it sprinkled all over the windup core code too.